src/new-client/src/Web/tearoutSections.ts

```diff
@@ -0,0 +1,47 @@
+import { bind } from "@react-rxjs/core"
+import { createSignal } from "@react-rxjs/utils"
+import { map } from "rxjs/operators"
+
+type TearOutEntry = [boolean, HTMLDivElement?]
+
+export const [tearOutEntryTiles$, tearOutTiles] = createSignal(
```

Comment:
Perhaps a better way to do this would be to have 1 signal for all the tear out windows and use a `type` to be able to distinguish what window it is. That way we can reduce the duplicated signals.
You could still bind this to create different hooks (useTearOutTileState, useTearOutAnalyticsState etc..) or just bind to one hook, whichever feels best.